ios/main.mm

```diff
@@ -200,13 +210,39 @@ void Vibrate(int mode) {
 
 int main(int argc, char *argv[])
 {
+	// Hacky hacks to try to enable JIT by pretending to be a debugger.
 	csops = reinterpret_cast<decltype(csops)>(dlsym(dlopen(nullptr, RTLD_LAZY), "csops"));
 	exc_server = reinterpret_cast<decltype(exc_server)>(dlsym(dlopen(NULL, RTLD_LAZY), "exc_server"));
 	ptrace = reinterpret_cast<decltype(ptrace)>(dlsym(dlopen(NULL, RTLD_LAZY), "ptrace"));
 	// see https://github.com/hrydgard/ppsspp/issues/11905
-	if (!cs_debugged()) {
+
+	// Tried checking for JIT support here with AllocateExecutableMemory and ProtectMemoryPages,
+	// but it just succeeds, and then fails when you try to execute from it.
+
+	// So, we'll just resort to a version check.
+
+	std::string version = [[UIDevice currentDevice].systemVersion UTF8String];
+	if (2 != sscanf(version.c_str(), "%d.%d", &g_iosVersionMajor, &g_iosVersionMinor)) {
+		// Just set it to 14.0 if the parsing fails for whatever reason.
+		g_iosVersionMajor = 14;
+		g_iosVersionMinor = 0;
+	}
+
+	if (g_iosVersionMajor > 14 || (g_iosVersionMajor == 14 && g_iosVersionMinor >= 3)) {
+		g_jitAvailable = false;
+	} else {
+		g_jitAvailable = true;
+	}
+
+	bool debuggerAttached = cs_debugged();
+
+	if (!debuggerAttached) {
```

Comment:
Hm, maybe better:

```c++
	if (debuggerAttached) {
		// Debugger is attached - we can actually JIT. Override the version detect.
		g_jitAvailable = true;
	} else if (g_jitAvailable) {
		// Try to fool iOS into thinking a debugger is attached.
		// This doesn't work in iOS 14.4 anymore.
		...
	}
```

That way we won't even try the hack anymore on newer iOS, which would presumably fix #9186 on those versions.

-[Unknown]

Reply:
Although that also means that there'd be no use for leaving the option available to set "just to see if it works on this new ios version", this hack will presumably still be required...

I'll just do this for 14.4 which we 100% know doesn't work.